examples/Dockerfile

```diff
@@ -14,7 +15,8 @@ ARG LOC=eidolon_examples
 FROM examples
 WORKDIR /app
 COPY $LOC .
-COPY demo_resources/ resources/
+COPY demo_resources/usage_resources.yaml resources/
+COPY eidolon_examples/conversational_chatbot/resources/conversational_agent.yaml /app/resources/conversational_agent.yaml
```

Comment:
I saw that you are mounting this file in the docker compose, is this addition still needed?

Reply:
Since we're using the SDK Dockerfile I just reverted all the changes I made to the examples Dockerfile